resources/js/components/SidebarNav.vue

```diff
@@ -0,0 +1,15 @@
+<script>
+export default {
+    data() {
+        return {
+            is_visible: screen.width <= 640 ? false : true,
+        };
+    },
+    created() {
+        const app = this;
+        Events.$on('openSidebarNav', function() {
```

Comment:
Instead of storing the `this` reference in another variable, use arrow functions.
```
Events.$on('openSidebarNav', () => this.isVisible = true);
```